logo-ann/benchmark/main.py

```diff
@@ -304,7 +308,7 @@ def evaluate():
             # print(f"Labels shape: {labels.shape}")
             with torch.inference_mode():
                 distance_matrix = (
-                    pairwise_squared_euclidian_distance(embeddings).cpu().numpy()
+                    pairwise_squared_euclidian_distance(embeddings).cpu().numpy() #you can change here the distance used
```

Comment:
It would have been good to have an argparse option to change the distance used instead of updating it in the code